### ISSUE TITLE
Missing Segment methods

### ISSUE THREAD

**Issue by nthtran** [expo/expo#143]
Such as `screen`, `group` and `alias` (see https://segment.com/docs/sources/mobile/android/).

**Comment by ctgardner** [comment]
Any progress on this issue? The `group` method is particularly useful.

**Comment by mathieumg** [comment]
`group` and `screen` seem there:

https://github.com/expo/expo/blob/ios/sdk24/android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/SegmentModule.java

**Comment by ctgardner** [comment]
@mathieumg Yep, I added them a while ago. Sorry, I should have updated this issue.

**Comment by raphaelrk** [comment]
Related to #998
